scm/driver/bitbucket/git_test.go

```diff
@@ -33,7 +33,10 @@ func TestGitFindCommit(t *testing.T) {
 
 	want := new(scm.Commit)
 	raw, _ := ioutil.ReadFile("testdata/commit.json.golden")
-	json.Unmarshal(raw, &want)
+	err = json.Unmarshal(raw, &want)
```

Comment:
Not required, by doing this ur returning the address of the pointer, which is not what we want.